### ISSUE TITLE
Is there any way to call a method when the job is buried ?

### ISSUE THREAD

**Issue by MhdSyrwan** [nesquena/backburner#97]
As mentioned [here](https://github.com/nesquena/backburner/blob/master/HOOKS.md), the `on_failure` hook is called when an exception is called (job failed) so if the `max_retry_count` was 3 it will be called 3 times. **Is there any way to call a method when the job is buried** ?

Thanks in advance

**Comment by contentfree** [comment]
@nesquena If you can pull in PR #104 then we should be able to close this.

@MhdSyrwan: Once the PR is pulled in you'll be able to define hooks like this:
```ruby
class MyJob
  # This hook is called only when the job is buried
  def self.on_bury_log(*args)
    # Do something
  end

  # This hook is called when the failed job is retried (and never when it's buried)
  def self.on_retry_log(retry_count, delay, *args)
    # Do something else
  end
end
```

**Comment by MhdSyrwan** [comment]
Great, Much Appreciated :+1:

**Comment by contentfree** [comment]
@nesquena This may be closed since #104 has been merged in.

**Comment by nesquena** [comment]
Thanks
